src/main/java/org/example/learningsystem/core/destination/service/DestinationService.java

```diff
@@ -0,0 +1,8 @@
+package org.example.learningsystem.core.destination.service;
+
+import org.example.learningsystem.email.config.EmailServerProperties;
+
+public interface DestinationService {
+
+    EmailServerProperties getEmailServerProperties();
```

Comment:
as DestinationService is an implementation for retriving any destination from SAP Destination Service, method name should be generic. It's also should return some generic response like Destination:
```
public Destination getByName(String name)
```